client/scss/components/_header.scss

```diff
@@ -49,7 +50,7 @@ header {
 
   .search-form .icon {
     @include svg-icon(1.3rem);
-    color: $nav-search-color;
+    color: color.adjust($color-white, $lightness: -20%);
```

Comment:
This variable wasn’t used anywhere else anymore, and it’s a bit silly to tie together the colors of icons across different parts of the UI.